qa/tasks/cephfs/filesystem.py

```diff
@@ -821,21 +821,15 @@ def create(self, **kwargs):
                 assert(subvols['create'] > 0)
 
                 self.run_ceph_cmd('fs', 'subvolumegroup', 'create', self.name, 'qa')
-                subvol_options = self.fs_config.get('subvol_options', '')
 
                 for sv in range(0, subvols['create']):
                     sv_name = f'sv_{sv}'
-                    cmd = [
-                      'fs',
-                      'subvolume',
-                      'create',
-                      self.name,
-                      sv_name,
-                      '--group_name', 'qa',
-                    ]
-                    if subvol_options:
-                        cmd.append(subvol_options)
-                    self.run_ceph_cmd(*cmd)
+                    svopts = self.fs_config.get('subvol_options', None)
+                    sv_create_cmd = ['fs', 'subvolume', 'create', self.name, sv_name]
+                    if svopts:
+                        sv_create_cmd.append(svopts)
+
+                    self.run_ceph_cmd(*sv_create_cmd)
```

Comment:
Besides replacing default from `''` to `None`, this commit also renames `cmd` to `sv_create_cmd`, changes writing style of definition of `sv_create_cmd` and rename `subvol_options` to `svopts`.

I think all these changes are not needed in this commit, simply replacing ` subvol_options = self.fs_config.get('subvol_options', '')` by ` subvol_options = self.fs_config.get('subvol_options', None)` should suffice.

Reply:
restored with minimal changes